.gitignore

```diff
@@ -1,3 +1,4 @@
+/.vscode/
```

Comment:
this kind of thing should go in your own global gitignore, not in every project you touch.